pkg/kubelet/sysctl/greylist.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sysctl
+
+import (
+	"strings"
+)
+
+// Namespace represents a kernel namespace name.
+type Namespace string
+
+const (
+	// the Linux IPC namespace
+	IpcNamespace = Namespace("ipc")
+
+	// the network namespace
+	NetNamespace = Namespace("net")
+
+	// the zero value if no namespace is known
+	UnknownNamespace = Namespace("")
+)
+
+var greylist = map[string]Namespace{
+	"kernel.sem": IpcNamespace,
+}
+
+var greylistPrefixes = map[string]Namespace{
+	"kernel.shm": IpcNamespace,
```

Comment:
Here in the greylist we should leave it, at least those are needed (maybe even `shmmni`):
```
kernel.shm_next_id = -1
kernel.shm_rmid_forced = 0
```